### PR TITLE
more benchmarks for perf monitoring

### PR DESCRIPTION
**Pull request by Kraks**

### CODE REVIEW

File: dev-clean/src/bench/scala/BenchLLSC.scala [hunk 1]
```diff
@@ -99,14 +115,15 @@ abstract class TestLLSC extends FunSuite {
...
     val TestPrg(m, name, f, config, cliArgOpt, exp) = tst
     test(name) {
       val code = llsc.newInstance(m, llsc.insName + "_" + name, f, config)
       code.genAll
       val mkRet = code.make(4)
       assert(mkRet == 0, "make failed")
-      for (i <- 1 to 10) {
-        Thread.sleep(5 * 1000)
+      for (i <- 1 to nTest) {
+        //Thread.sleep(5 * 1000) // GW: do we need that?
```
Comment (Tantalus13A98B5F) [inline dev-clean/src/bench/scala/BenchLLSC.scala:126]: we can make it shorter, 1 or 2s; better to keep it.

**Comment by Tantalus13A98B5F** [comment]
I wonder if we should test cases without caching.

also, I wonder how long would these cases take now?

File: dev-clean/build.sbt [hunk 2]
```diff
@@ -41,11 +41,12 @@ addCompilerPlugin("org.scala-lang.plugins" % "scala-continuations-plugin_2.12.0"
 addCompilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.0-M4")
 
 parallelExecution in Test := false
+parallelExecution in Bench := false
```
[the remark this answers is not in the document]
Comment (Kraks) [reply dev-clean/build.sbt:44]: right, I have not observed anything weird -- parallel execution should already be disabled.

**Comment by Kraks** [comment]
> I wonder if we should test cases without caching.

The cases tested with Z3 do not use any caching, so it should give us some info on that.

> also, I wonder how long would these cases take now?

Each of them takes some reasonable time, around 20+ sec iirc.

**Pull request merged**